docs/code/getting-started.md

```diff
@@ -417,54 +378,108 @@ Let's review all we've learned to do:
+        )
+
+    The `merge_file` method combines translating and installing files, similarly to how the `executable` method combines building and installing your app.
+
+3. Still in this file, add the following as the last line:
+
+        subdir('po')
+
+4. You might have noticed in step 2 that the `merge_file` method has an `input` and `output`. We're going to append the additional extension `.in` to our .desktop and .appdata.xml files so that this method can take the untranslated files and produce translated files with the correct names.
 
     ```bash
-    cmake -DCMAKE_INSTALL_PREFIX=/usr ../
-    make pot
+    git mv data/com.github.yourusername.yourrepositoryname.desktop data/com.github.yourusername.yourrepositoryname.desktop.in
+    git mv data/com.github.yourusername.yourrepositoryname.appdata.xml data/com.github.yourusername.yourrepositoryname.appdata.xml.in
     ```
 
-5. Don't forget to add this new directory and it's contents to git
+   We use the `git mv` command here instead of renaming in the file manager or with `mv` so that `git` can keep track of the file rename as part of our revision history.
+
+5. Now, Create a directory named "po" in the root folder of your project. Inside of your po directory you will need to create another meson.build file. This time, its contents will be:
+
+        i18n.gettext(meson.project_name(),
+          args: [
+            '--directory=' + meson.source_root(),
+            '--from-code=UTF-8'
+          ]
+        )
+
+6. Inside of "po" create another file called "POTFILES" that will contain paths to all of the files you want to translate. For us, this looks like:
+
+        src/Application.vala
+        data/com.github.yourusername.yourrepositoryname.desktop.in
+        data/com.github.yourusername.yourrepositoryname.appdata.xml.in
+
+7. We have one more file to create in the "po" directory. This file will be named "LINGUAS" and it should contain the two-letter language codes for all languages you want to provide translations for. As an example, let's add German and Spanish
+
+        de
+        es
+
+8. Now it's time to go back to your build directory and generate some new files! The first one is our translation template or `.pot` file:
+
+        ninja com.github.yourusername.yourrepositoryname-pot
+
+    After running this command you should notice a new file in the po directory containing all of the translatable strings for your app.
+
+9. Now we can use this template file to generate translation files for each of the languages we listed in the LINGUAS file with the following command:
+
+        ninja com.github.yourusername.yourrepositoryname-update-po
+
+    You should notice two new files in your po directory called `de.po` and `es.po`. These files are now ready for translaters to localize your app!
+
+10. Last step. Don't forget to add all of the new files we created in the po directory to git:
 
     ```bash
     git add po/
```

Comment:
I suppose we have to add `src/Application.vala`, `meson.build`, and `data` directory.
That is to say, `git add po/ src/Application.vala meson data/`